packages/algorithm/package.json

```diff
@@ -39,6 +38,10 @@
     "test:ie": "cd tests && karma start --browsers=IE",
     "watch": "tsc --build --watch"
   },
+  "typedoc": {
+    "entryPoint": "./src/index.ts",
+    "displayName": "algorithm"
```

Comment:
Without setting display name like this, the breadcrumbs in the rendered docs end up looking like: `@lumino / @lumino/algorithm / member`. You'll have to check the rendered docs, and see for yourself if you like it prefer it this way or not.